api_tests/tests/e2e/bitcoin_ethereum.ts

```diff
@@ -0,0 +1,455 @@
+/**
```

Comment:
Oh I see, ignore my `git mv` comment then.

Reply:
GitHub tells me this file is too large to display. I think we should listen to GitHub and split it.

Reply:
Good point. 
Unfortunately with this setup it is not possible. 
This has additionally the downside that the tests in this file are not run in parallel. 
In order to allow for splitting this up we would need to change something about how we setup things. This however would be (again) a huge change. 
In the PR description I proposed to do this in a follow up ticket.